src/components/Icons/Arrows/ArrowChevronDoubleDown.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Path } from 'react-native-svg';
+import { colors } from '@magnetis/astro-galaxy-tokens';
+import { IconProps } from '../types';
+import BaseIcon from '../BaseIcon';
+import { getFill } from '../utils';
```

Comment:
:wrench: You can organize these imports

```suggestion
import React from 'react';
import { Path } from 'react-native-svg';
import { colors } from '@magnetis/astro-galaxy-tokens';

import BaseIcon from '../BaseIcon';
import { getFill } from '../utils';

import type { IconProps } from '../types';
```